.github/ISSUE_TEMPLATE/new_press.yml

```diff
@@ -0,0 +1,76 @@
+name: Request to add a new press release or press link
+description: Use this issue to request to add a new press release or press link to Calitp.org
+title: "New press request"
+labels:
+  - content
+  - press
+body:
+  - type: input
+    id: title
+    attributes:
+      label: Title
+      description: The title is used in list views on the home page and [https://calitp.org/press](https://calitp.org/press)
+    validations:
+      required: true
+  - type: input
+    id: heading
+    attributes:
+      label: Heading (optional)
+      description: Optional heading if different from the title, used on the page for this press release
+  - type: input
+    id: link
+    attributes:
+      label: Asset URL
+      description: "The URL of the resource: link to Google Drive content of press release, or external link to a press item outside of Cal-ITP"
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: Tag
+      description: What tag is this press item in?
+      multiple: false
+      options:
+        - Contactless payments
+        - Benefits
+        - GTFS
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: Date - Year
+      description: The date is used for press releases. Not required for external press links.
+      multiple: false
+      options:
+        - 2023
+        - 2022
+        - 2021
+        - 2020
+  - type: dropdown
+    attributes:
+      label: Date - Month
+      description: The date is used for press releases. Not required for external press links.
+      multiple: false
+      options:
+        - "01"
+        - "02"
+        - "03"
+        - "04"
+        - "05"
+        - "06"
+        - "07"
+        - "08"
+        - "09"
+        - "10"
+        - "11"
+        - "12"
+  - type: input
+    id: date
+    attributes:
+      label: Date
+      description: The date is used for press releases. Not required for external press links.
```

Comment:
Since there is a separate Year and Month field, I think this is duplicative right?